bucket/ilspy.json

```diff
@@ -14,9 +14,9 @@
     ],
     "checkver": {
         "github": "https://github.com/icsharpcode/ILSpy",
-        "regex": "v[\\d.]+/ILSpy_binaries_([\\d.]+)\\.zip"
+        "regex": "v(?<tag>[\\d.]+)/ILSpy_binaries_(?<version>[\\d.]+)\\.zip"
     },
     "autoupdate": {
-        "url": "https://github.com/icsharpcode/ILSpy/releases/download/v$matchHead/ILSpy_binaries_$version.zip"
+        "url": "https://github.com/icsharpcode/ILSpy/releases/download/v$matchTag/ILSpy_binaries_$version.zip"
```

Comment:
```suggestion
        "url": "https://github.com/icsharpcode/ILSpy/releases/download/v$majorVersion.$minorVersion/ILSpy_binaries_$version.zip"
```

Reply:
![image](https://user-images.githubusercontent.com/6400093/86886159-c58efb00-c139-11ea-880a-8dd07101e8c8.png)


The tag is only major.minor when there is no patch version, when there is a patch version is it major.minor.patch.

Reply:
Then it will be changed for next version